src/plugins/system/plug_system_tc/plug_system_tc.php

```diff
@@ -15,6 +15,8 @@
 jimport('joomla.html.parameter');
 jimport('joomla.plugin.plugin');
 
+$lang = JFactory::getLanguage();
+$lang->load('plug_system_tc', JPATH_ADMINISTRATOR);
```

Comment:
use this as class var instead

```
	/**
	 * Load plugin language file automatically so that it can be used inside component
	 *
	 * @var    boolean
	 * @since  2.3.4
	 */
	protected $autoloadLanguage = true;
```